docs/source/platform/client-awareness.md

```diff
@@ -1,4 +1,117 @@
 ---
 title: Identifying clients
-description: How to configure Apollo Engine for client awareness
----
\ No newline at end of file
+description: What is client awareness and how to add it to the Apollo Platform
+---
+
+Client identity is central to the Apollo Platform and enables tracking how all
+the consumers use the data graph. The Apollo Platform allows segmenting usage
+data by client name and version. Filtering by client information provides a
+field-level understanding of how the users interact with the GraphQL api in
+real-time. In addition to per client metrics, understanding this granular
+detail informs how the GraphQL schema can evolve and reacts to a new client
+release.
+
+## Use Cases
+
+Often a GraphQL api is used by multiple consumers with different frequencies,
+subselections, and permissions. The Apollo Platform accepts client name and
+version, tracking these requests across the stack and enabling filtering on
```

Comment:
This sentence sounds a little strange to me. Maybe something like "The Apollo Platform allows specification of client name and version with all reported metrics to enable filtering those reported metrics to a specific client or set of clients"
don't need to add, but across different stacks as well :O holy moly!